### DISCUSSION TITLE
Is there a way to select only files that are in the same folder under different name?

### DISCUSSION THREAD

**Discussion by Shajirr** [qarmin/czkawka#1271] [Q&A]
The folders being compared contain subfolders. 
Some of the duplicate files have exact same path, being in the same subfolder, under different names. 
I want to select only these ones, to delete the duplicates, but I don't see how.

What I don't want to select are all other duplicates, which have different paths, as I want to make hardlinks of them instead.

Searching each final folder individually is not an option, too many of them.

**Comment by killo3967** [comment]
Yes, i want the same. I want to find duplicate files that i have in my hdd but has diferent names, diferent extension and diferent sizes but internally are the same, and tey are in the same disk.
